pageserver/src/billing_metrics.rs

```diff
@@ -279,3 +308,36 @@ pub async fn collect_metrics_task(
 
     Ok(())
 }
+
+/// Caclculate synthetic size for each active tenant
+pub async fn calculate_synthetic_size_worker() -> anyhow::Result<()> {
+    info!("starting calculate_synthetic_size_worker");
+
+    let calculate_synthetic_size_interval = Duration::from_secs(10);
```

Comment:
you had a constant for another worker. Hope you'll follow the same approach here